bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/runtime/ShellExecutorHelper.java

```diff
@@ -273,4 +278,30 @@ protected void processLine(String s, int i) {
             SLogger.log(LogI.INFO, "SERVER_SER_MSG >> " + s);
         }
     }
+
```

Comment:
The idea for this method is from here: https://stackoverflow.com/a/45203609/8149485

Reply:
please add the link into the source code comment.